docs/install_guides/pypi-openvino-dev.md

```diff
@@ -0,0 +1,67 @@
+
+**Included with the Installation and installed by default:**
+
+| Component                                                                                           | Description                                                                                                                                                                                                                                                                                                   |  
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Model Optimizer](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) | This tool imports, converts, and optimizes models that were trained in popular frameworks to a format usable by Intel tools, especially the Inference Engine. <br>Popular frameworks include Caffe\*, TensorFlow\*, MXNet\*, and ONNX\*.                                                                              |
+| Additional Tools                                   | A set of tools to work with your models including [Accuracy Checker utility](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker_README.html), [Post-Training Optimization Tool](https://docs.openvinotoolkit.org/latest/pot_README.html)  |
+
+**Installed by dependency:**
+
+| Component                                                                                           | Description                                                                                                                                                                                                                                                                                                   |  
+|-----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Inference Engine](https://pypi.org/project/openvino)               | This is the engine that runs the deep learning model. It includes a set of libraries for an easy inference integration into your applications.                                                                                                                                                                |
+
+
+## Supported Operating Systems
```

Comment:
I guess, @ryanloney suggested changing not this line from `System Requirements` to `Supported Operating Systems`, but the line below: `Operating Systems` to `Supported Operating Systems`. Is it right?

Reply:
Yes, my bad, fixed